docs/conf.py

```diff
@@ -0,0 +1,33 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+
+project = 'QE Metrics'
+copyright = '2023, ilausuch'
+author = 'ilausuch'
```

Comment:
Could be more than one author? will be others contributors

Reply:
yes

Reply:
but i put just you as it is mainly your modules so far